src/main/kotlin/io/emeraldpay/dshackle/upstream/calls/DefaultEthereumMethods.kt

```diff
@@ -127,20 +132,22 @@ class DefaultEthereumMethods(
     init {
         allowedMethods = anyResponseMethods +
             firstValueMethods +
+            possibleNotIndexedMethods +
             specialMethods +
             headVerifiedMethods -
             chainUnsupportedMethods(chain) +
             getChainSpecificMethods(chain)
     }
 
-    override fun createQuorumFor(method: String): CallQuorum {
+    override fun createQuorumFor(method: String, multistream: Multistream): CallQuorum {
         return when {
             newFilterMethods.contains(method) -> NotLaggingQuorum(4)
             withFilterIdMethods.contains(method) -> AlwaysQuorum()
             hardcodedMethods.contains(method) -> AlwaysQuorum()
             firstValueMethods.contains(method) -> AlwaysQuorum()
             anyResponseMethods.contains(method) -> NotLaggingQuorum(4)
             headVerifiedMethods.contains(method) -> NotLaggingQuorum(1)
+            possibleNotIndexedMethods.contains(method) -> NotNullQuorum(multistream.getAll().filter { it.isAvailable() }.size)
```

Comment:
In general case size of list of available upstreams can differ from size of upstreams loaded to FilteredAPIs. For example - upstream can change status, matcher can be applied and so on. Maybe we can just fix number of retries.

Reply:
I don't like fixed tries because it doesn't represent anything, for my opinion. We want to iterate over all appropriate upstreams and as you said their size may differ from time to time so fixed tries don't help to achieve that

Reply:
Done